### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by ghost** [lukeaschenbrenner/TxtNet-Browser#31]
Are there any plans on publishing TxtNet-Browser to F-Droid?

**Comment by lukeaschenbrenner** [comment]
Yes, I'd like to get this project on the repo. There are still a few long-standing bugs that I'll be taking the time to fix over this month, and then I'll need to work on making reproducible builds for the custom Brotli4J build because I don't think F-Droid accepts .aar libraries.
